ktor-io/common/src/io/ktor/utils/io/ByteWriteChannelOperations.kt

```diff
@@ -30,6 +30,28 @@ public suspend fun ByteWriteChannel.writeInt(value: Int) {
     flushIfNeeded()
 }
 
+/**
+ * Writes a 32-bit floating-point value to the current [ByteWriteChannel].
+ *
+ * @param value The floating-point value to be written to the channel.
```

Comment:
Probably, could be shortened
```suggestion
 * Writes a 32-bit floating-point [value] to the current [ByteWriteChannel].
```